lib/atom-cfn-lint.js

```diff
@@ -45,6 +45,11 @@ export default {
   // activate linter
   activate() {
     const helpers = require('atom-linter');
+
+    require('atom-package-deps').install('atom-cfn-lint')
+      .then(function() {
+        console.log('All dependencies installed, good to go.')
```

Comment:
I would prepend your package name so users can identify where this line is coming from. You _might_ also wrap it in a check on [`Atom::inDevMode()`](https://atom.io/docs/api/v1.27.2/AtomEnvironment#instance-inDevMode) to only show this when Atom was started with `--dev`.